services/storage/src/simcore_service_storage/handlers_files.py

```diff
@@ -152,6 +152,8 @@ async def upload_file(request: web.Request) -> web.Response:
     """creates upload file links:
 
     This function covers v1 and v2 versions of the handler.
+    Note: calling this entrypoint on an already existing file will overwrite that file. That file will be deleted
+    before the upload takes place.
```

Comment:
This means, that from now onwards we shall see a delete mark each time a file changes in the output ports?

Reply:
I wonder how this impacts dynamic services, if it impacts them at all.